### PR TITLE
Homework4: Igor Bohdan

### PR DESCRIPTION
**Pull request by IgorBohdan3407**

### CODE REVIEW

File: task2.city.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Moscow
+# Novgorod
+# #########
+# Создаем словарь с ключом - страна, значение - города(список),
+# получаем список городов для проверки, пробегаемся по нему,
+# сверяем значение словаря со списком для проверки, если
+# находим печатаем ключ.
+my_dict = {}
+for i in range(int(input())):
+    country, *city = input().split()
```
Comment (belovmd) [inline task2.city.py:25]: good!

File: task2.city.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    my_dict[country] = city  # создаем словарь
+list_check = []
+for j in range(int(input())):
+    city = input()
+    list_check.append(city)  # сщздаем список для проверки
```
Comment (belovmd) [inline task2.city.py]: misprint in comment :)

File: task2.city.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Дан список стран и городов каждой страны.
```
Comment (belovmd) [inline task2.city.py:1]: what about Brest in Belarus and France?

File: task4.different.numbers.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Даны два списка чисел.
```
Comment (belovmd) [inline task4.different.numbers.py:1]: simmetric difference is a one-line solution for this task

File: task7.py [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    if a > b:
+        a %= b
+    else:
+        b %= a
+# т.к. заранее не известно в какой
+# переменной сохранится результат
+# складываем переменные, так как
+# во второй все равно в конце
+# цилкба будет 0
+most_overall_divider = a + b
```
Comment (belovmd) [inline task7.py]: `gcf = a if a else b. `
you write a cheat.

**Review by belovmd — COMMENTED** [review]
please address next issues:

File: task2.city.py (hunk 1 above)
Comment (IgorBohdan3407) [reply task2.city.py:25]: thanks.

File: task2.city.py (hunk 2 above)
Comment (IgorBohdan3407) [reply task2.city.py]: yes, I know, thank you)

File: task2.city.py (hunk 3 above)
Comment (IgorBohdan3407) [reply task2.city.py:1]: should work, I checked

File: task7.py (hunk 5 above)
Comment (IgorBohdan3407) [reply task7.py]: What exactly is a cheat? The code is working.

**Review by belovmd — COMMENTED** [review]
Please address previous review issues

**Pull request closed without merging**